app/Models/NetworkHost.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Adshares\Adserver\Models;
+
+use Adshares\Adserver\Models\Traits\AccountAddress;
+use Adshares\Adserver\Models\Traits\AutomateMutators;
+
+use Illuminate\Database\Eloquent\Model;
+
+class NetworkHost extends Model
+{
+    use AccountAddress;
+    use AutomateMutators;
+
+    protected $primaryKey = 'address';
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "address" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1192)